recipe/meta.yaml

```diff
@@ -1,60 +1,71 @@
-    folder: menuinst_src                                                                    # [win]
-    patches:                                                                                # [win]
-      - menuinst_patches/0003-root-prefix-to-prefix.patch                                   # [win]
-      - menuinst_patches/0004-distribution-name-subst.patch                                 # [win]
+      - "{{ SRC_DIR | replace('\\', '/') }}/src/conda_patches/0001-Rename-and-replace-entrypoint-stub-exe.patch"
+      - "{{ SRC_DIR | replace('\\', '/') }}/src/conda_patches/0002-Manipulate-PATH-directly-instead-of-_call_ing-conda.patch"
+      - "{{ SRC_DIR | replace('\\', '/') }}/src/conda_patches/0003-Return-unknown-module-in-deprecations.patch"
   - url: https://github.com/conda/constructor/archive/{{ constructor_version }}.tar.gz  # [win]
-    sha256: b000dde0ac8641c92f7342362173968505d4e85845be37655b5b9221cb65f3a8  # [win]
-    folder: constructor  # [win]
+    sha256: 787ffd85e9414bdf70fe531f01eab3987a040e3f6a6ac3a01409f4d332f7de9e  # [win]
+    folder: constructor_src  # [win]
 
 build:
-  number: 1
+  number: 0
+  ignore_run_exports:
+    - '*'
 
 requirements:
   build:
-    - python {{ python_version }}
-    - pyinstaller 5.6.2
-    - conda {{ version }}
-    - conda-package-handling >=1.7.2
-    - patch  # [not win]
-    - m2-patch  # [win]
+    - pyinstaller
+    - python ={{ python_version }}
+    - conda ={{ version }}
+    - conda-package-handling >=1.6
+    - menuinst >=1.4.18,<2.0a0  # [win]
+    - conda-libmamba-solver ={{ conda_libmamba_solver_version }}
+    - libmambapy ={{ libmambapy_version }}
   run_constrained:
     - constructor >={{ constructor_version }}
 
 test:
+  requires:
+    - pytest
+  source_files:
+    - tests
   commands:
-    - "${PREFIX}/standalone_conda/conda.exe -V"  # [unix]
-    - "${PREFIX}/standalone_conda/conda.exe create -y -p ./env_test zlib tqdm" # [unix]
-    - "%PREFIX%\\standalone_conda\\conda.exe -V"  # [win]
-    - "%PREFIX%\\standalone_conda\\conda.exe create -y -p env_test zlib tqdm"  # [win]
+    - pytest -vv # [not s390x]
+    - pytest -vv -k "not test_new_environment" # [s390x]
 
 about:
-  home: https://github.com/AnacondaRecipes/conda-standalone-feedstock
-  dev_url: https://github.com/AnacondaRecipes/conda-standalone-feedstock
-  doc_url: https://github.com/AnacondaRecipes/conda-standalone-feedstock/blob/master/README.md
-  license: BSD-3-Clause
-  license_family: BSD
-  license_file: LICENSE
+  home: https://github.com/conda/conda-standalone
   summary: Entry point and dependency collection for PyInstaller-based standalone conda.
-  description: Standalone version of conda to bootstrap conda for Miniconda and Anaconda Distribution.
+  description: A standalone conda executable built with PyInstaller.
+  license: LicenseRef-CondaStandalone
+  license_file:
+    - LICENSE
+    - 3rd-party-licenses.json
+  license_family: BSD
```

Comment:
This is incorrect. There is no license family because there are multiple licenses. Use `skip-lint` to skip that check